packages/core/src/Modules/Endpoint/FeatureFlagsSection.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { observer, useStore } from '@deriv/stores';
+import { website_domain, TRADE_FEATURE_FLAGS } from '@deriv/shared';
+import { Checkbox, Text } from '@deriv/components';
+
+export const FeatureFlagsSection = observer(() => {
+    const { feature_flags } = useStore();
+    const HIDDEN_FEATURE_FLAGS = ['wallet'];
+
+    const visible_feature_flags = Object.entries(feature_flags.data ?? {})?.reduce<{ [key: string]: boolean }>(
+        (flags, [key, value]) => {
+            const is_production = location.hostname === website_domain;
+            if ((!is_production || !TRADE_FEATURE_FLAGS.includes(key)) && !HIDDEN_FEATURE_FLAGS.includes(key)) {
+                flags[key] = value;
+            }
+            return flags;
+        },
+        {} // hiding trade features flags from production
+    );
+    if (!feature_flags.data) return null;
```

Comment:
Maybe for checking objects it is better to use `isEmptyObject`? Because they are not falsy
Also this check can be lifted up before `const visible_feature_flags` I guess